src/components/AvatarButton.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { oneOfType, oneOf, string, number } from 'prop-types'
+import glamorous from 'glamorous'
+
+import { colors, transition, radii } from '../theme'
+import { toAlpha } from '../utils/style'
+import Avatar from './Avatar'
+
+const UnstyledButton = glamorous.button(props => ({
+  display: 'inline-block',
+  margin: 0,
+  padding: 0,
+  lineHeight: 0,
+  background: 'transparent',
+  border: 0,
+  outline: 0,
+  cursor: 'pointer',
+  borderRadius: props.shape === 'square' ? radii[0] : '50%',
```

Comment:
Also, is it a little weird to call it an `UnstyledButton` when it has at least some style applied to it? i.e. the borderRadius here isn't really just overriding/resetting default styles.